src/main/java/edu/greenblitz/robotName/subsystems/Intake/NeoIntake.java

```diff
@@ -25,5 +28,6 @@ public void updateInputs(IntakeInputsAutoLogged chassisIntakeInputs) {
 		chassisIntakeInputs.outputCurrent = motor.getOutputCurrent();
 		chassisIntakeInputs.appliedOutput = motor.getAppliedOutput();
 		chassisIntakeInputs.velocity = motor.getEncoder().getVelocity();
+		chassisIntakeInputs.beamBreakerValue = beamBreaker.get();
```

Comment:
why is the name chassisIntakeIInputs and not just intakeInputs

Reply:
Thank you Romy for your helpful opinion!!! -noam <3